content/03.results.md

```diff
@@ -51,47 +52,60 @@ Using real reads, the three HG00514, HG00733, and NA19240 individuals were teste
 GiaB: Real reads from the HG002 individual were used to genotype SVs and compared with the high-quality calls from the Genome in a Bottle consortium[@doi:10.1038/sdata.2016.25;@doi:10.1101/281006].
 Maximum F1 score for each method (color), across the whole genome or focusing on non-repeat regions (x-axis). 
 We evaluated the ability to predict the presence of an SV (transparent bars) and the exact genotype (solid bars).
-SVTyper cannot genotype insertions hence the absent bars in the top panel.
+Results are separated across panels by variant type: insertions and deletions.
+SVTyper cannot genotype insertions, hence the missing bars in the top panels.
 ](images/hgsvc-giab-best-f1.png){#fig:2}
 
+[//]: # (The caption in the figure says that the real read experiment included HG00514, HG00733, and NA19240. Here you say only HG00514. You should make sure that whichever one is incorrect is fixed up.)
 We then repeated the analysis using real Illumina reads from HG00514 to benchmark the methods on a more realistic experiment.
-Here vg clearly outperformed other approaches, most likely because of its graph-based strategy and robustness to errors in the SV catalog (Figures {@fig:2} and  {@fig:hgsvc-real-geno}).
-In non-repeat regions and across the whole genome, the F1 scores and precision-recall AUC were higher for vg compared to other methods. 
+Here vg clearly outperformed other approaches (Figures {@fig:2} and  {@fig:hgsvc-real-geno}).
+In non-repeat regions and across the whole genome, the F1 scores and precision-recall AUC (data not shown) were higher for vg compared to other methods. 
```

Comment:
fig-S4 referenced in prev. sentence shows higher AUC.

Reply:
ah, my bad